src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <article class="card" data-qa="card">
+      <div class="card__photo">
+        <img
+          src="images/imac.jpeg"
+          alt="Apple A1419 iMac 27"
+          class="card__img"
+        >
+      </div>
```

Comment:
This div is redundant
```suggestion
      <img
        src="images/imac.jpeg"
        alt="Apple A1419 iMac 27"
        class="card__img"
      >
```